invenio_base/app.py

```diff
@@ -244,6 +257,19 @@ def loader_init_func(bp_or_func):
     _loader(app, loader_init_func, entry_points=entry_points, modules=modules)
 
 
+def urls_builder_loader(app, factory, **kwargs):
+    """Loads the urls builder.
+
+    :param app: current Flask App
+    :param factory: callable ``(Flask.App, **kwargs) -> InvenioURLsBuilder``
+    """
+    app.add_template_global(invenio_url_for)
+    if factory:
+        app._urls_builder = factory(app, **kwargs)
+    else:
+        app._urls_builder = NoOpInvenioUrlsBuilder()
```

Comment:
Just to have a default... although I am not sure if in the end it helps to have a generic default :shrug: